MyBooking.Repositories/MyBooking.Repositories.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
What does this file do?

Reply:
A software development kit (SDK) is a collection of software development tools in one installable package.

Reply:
Do we actually need it in the repo?

Reply:
I think it’s possible to simplify the launch of the project.

Reply:
Discussed